packages/sdk/contractkit/src/wrappers/Escrow.ts

```diff
@@ -1,25 +1,151 @@
+   * @return An array containing all the IDs of the Escrowed Payments that were sent by the
+   * specified sender.
+   */
   sentPaymentIds = proxyCall(this.contract.methods.sentPaymentIds)
 
+  /**
+   * @notice Gets array of all Escrowed Payments received by identifier.
+   * @param identifier The hash of an identifier of the receiver of the escrowed payment.
+   * @return An array containing all the IDs of the Escrowed Payments that were received
+   * by the specified receiver.
+   */
   getReceivedPaymentIds = proxyCall(this.contract.methods.getReceivedPaymentIds)
 
+  /**
+   * @notice Gets array of all trusted issuers set per paymentId.
+   * @param paymentId The ID of the payment to get.
+   * @return An array of addresses of trusted issuers set for an escrowed payment.
+   */
   getSentPaymentIds = proxyCall(this.contract.methods.getSentPaymentIds)
 
-  transfer = proxySend(this.connection, this.contract.methods.transfer)
+  /**
+   * @notice Gets trusted issuers set as default for payments by `transfer` function.
+   * @return An array of addresses of trusted issuers.
+   */
+  getDefaultTrustedIssuers = proxyCall(this.contract.methods.getDefaultTrustedIssuers)
+
+  /**
+   * @notice Gets array of all trusted issuers set per paymentId.
+   * @param paymentId The ID of the payment to get.
+   * @return An array of addresses of trusted issuers set for an escrowed payment.
+   */
+  getTrustedIssuersPerPayment = proxyCall(this.contract.methods.getTrustedIssuersPerPayment)
+
+  /**
+   * @notice Transfer tokens to a specific user. Supports both identity with privacy (an empty
+   *         identifier and 0 minAttestations) and without (with identifier and minAttestations).
+   *         Sets trustedIssuers to the issuers listed in `defaultTrustedIssuers`.
+   *         (To override this and set custom trusted issuers, use `transferWithTrustedIssuers`.)
+   * @param identifier The hashed identifier of a user to transfer to.
+   * @param token The token to be transferred.
+   * @param value The amount to be transferred.
+   * @param expirySeconds The number of seconds before the sender can revoke the payment.
+   * @param paymentId The address of the temporary wallet associated with this payment. Users must
+   *        prove ownership of the corresponding private key to withdraw from escrow.
+   * @param minAttestations The min number of attestations required to withdraw the payment.
+   * @return True if transfer succeeded.
+   * @dev Throws if 'token' or 'value' is 0.
+   * @dev Throws if identifier is null and minAttestations > 0.
+   * @dev If minAttestations is 0, trustedIssuers will be set to empty list.
+   * @dev msg.sender needs to have already approved this contract to transfer
+   */
+  transfer: (
+    identifier: Identifier,
+    token: Address,
+    value: number | string,
+    expirySeconds: number | string,
+    paymentId: string,
+    minAttestations: number | string
```

Comment:
`value`, `expirySeconds`, and `minAttestations` are always numbers, and `paymentId` is an address